packages/rum/test/rum.spec.ts

```diff
@@ -665,4 +673,26 @@ describe('rum event assembly', () => {
     expect(getRumMessage(server, 0).view.url).toEqual('http://foo.com/')
     expect(getRumMessage(server, 1).view.url).toEqual('http://foo.com/bar')
   })
+
+  it('should keep the same URL when updating an ended view', () => {
+    const { server, lifeCycle, clock } = setupBuilder
+      .withFakeClock()
+      .withFakeLocation('http://foo.com/')
+      .build()
+
+    clock.tick(VIEW_DURATION)
```

Comment:
Why do we need to set a view duration to make the test work?

Reply:
If we don't advance the time, then the first view and the second view all starts at the same time. Since the `parentContext` is using the `startTime` to match an view event with its view context, the initial view update is incorrectly using the `currentContext` (second view context) https://github.com/DataDog/browser-sdk/blob/master/packages/rum/src/parentContexts.ts#L135